docs/dev/guide/l2-l1.md

```diff
@@ -1,3 +1,42 @@
 # L2 -> L1 communication
 
-Sending messages from zkSync to Ethereum is not yet available, but it will be implemented later.
+This section describes the interface for interaction with Ethereum from L2. It assumes that you are already familiar with the basic concepts of working with the L2 -> L1 communication. If you are new to this topic, you can read the conceptual introduction [here](../zksync-v2/l1-l2-interop.md). 
+
+## Structure
+
+Unlike the L1 -> L2 communication, it is impossible to directly initialize transactions from L2 to L1. However, you can send an arbitrary-length message from zkSync to Ethereum and then handle the received message on L1 smart contract. From the L2 side, to send a message special system contract should be called. It accepts only bytes of the message that will be sent to the Ethereum `zkSync` smart contract. From the L1 side, the `zkSync` smart contract provides an interface to prove that message was sent to L1 in a certain zkSync block. 
+
+## Sending a message on L2
```

Comment:
You should mention that to send the message the L1Messenger system contract should be called (to the interface/address you can just directly link the section that I already uploaded in my branch)